contracts/estate/EstateRegistry.sol

```diff
@@ -372,4 +372,26 @@ contract EstateRegistry is Migratable, ERC721Token, ERC721Receiver, Ownable, IEs
 
     return uint256(out);
   }
+
+  /**
+   * @dev update LAND data owned by an Estate
+   * @param estateId Estate
+   * @param x coordinate of the LAND
+   * @param y coordinate of the LAND
+   * @param data string metadata
+   */
+  function updateLandData(uint256 estateId, int x, int y, string data) external onlyUpdateAuthorized(estateId) {
```

Comment:
Question:
I don't get why are we requiring `estateId` here. Because it's an argument, the caller could supply an estateId `X` that doesn't have anything to do with `x,y`.

(same for `updateManyLandData`)

Reply:
Yes, it is related of what @eordano said. I need to use it also to check if the parcels are part of that estate